docs/guide/models/execution.rst

```diff
@@ -0,0 +1,29 @@
+Protocols and the Execution Model
+=================================
+
+Protocols in OpenFE are built on a flexible execution model. 
+Result objects are shaped by this model, and therefore some basic
+background on it can be useful when looking into the details of simulation
+results, even if most users don't need to work with the details of the
+execution model.
+
+Each protocol involves a number of steps (called ``ProtocolUnit``\ s) which occur in
+some order. Formally, this is described by a directed acyclic graph (DAG),
+so the collection of steps to run is called a ``ProtocolDAG``. A
+:class:`.Protocol` creates the ``ProtocolDAG``, and a single ``ProtocolDAG``
+should give information necessary to obtain an estimate (perhaps a very poor
```

Comment:
I'd remove the "perhaps a very poor estimate" here, I get what you mean (especially in the context of NEQ), but it's probably more distracting than anything in this context.